src/style.css

```diff
@@ -0,0 +1,123 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;700&family=Inter:wght@300;400;700;800&family=Roboto+Condensed:ital,wght@1,300;1,400;1,700&family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1,
+p {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background-color: #fff;
+  border-radius: 5px;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  max-width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars-block {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+}
+
+.stars-block__star {
+  background-image: url("./images/star.svg");
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars-block--stars-4 .stars-block__star:nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
Due to the task description
![image](https://user-images.githubusercontent.com/80700247/230380129-11254c09-9189-4c3d-a546-d102ac67c0cf.png)